util/arena.h

```diff
@@ -79,6 +79,12 @@ class Arena : public Allocator {
 
   size_t BlockSize() const override { return kBlockSize; }
 
+  bool IsInInlineBlock() const {
+    assert(blocks_memory_ == sizeof(inline_block_) ||
+           blocks_memory_ >= kBlockSize);
+    return blocks_memory_ == sizeof(inline_block_);
```

Comment:
Can you do `blocks_.size() == 0` instead?

I'm worried that if arena size is sizeof(inline_block_), this won't work.